NitroxServer-Subnautica/Program.cs

```diff
@@ -73,7 +76,10 @@ private static async Task Main(string[] args)
                 {
                     throw new Exception("Unable to start server.");
                 }
-                Log.Info("Server is waiting for players!");
+                watch.Stop();
+
+                Log.Info($"Server started ({Math.Round(watch.Elapsed.TotalSeconds, 1)}s)");
```

Comment:
I agree with some text changes here as to actually saying the server has started but I'm not sure of the value of having a timer and telling them how long it took, will this be useful?

Reply:
Imo, time metrics are important when you're dealing with servers, always seen those kind of informations when launching game servers, but I can remove it, if no one else find it useful